src/main/java/com/project/springbootproject/service/BookServiceImpl.java

```diff
@@ -38,4 +41,20 @@ public BookDto findBookById(Long id) {
         );
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public void updateBook(CreateBookRequestDto bookDto, Long id) {
```

Comment:
If you use this dto for both creation and updating, let's rename it to just `BookRequestDto`